### DISCUSSION TITLE
Tshock 1.4.1?

### DISCUSSION THREAD

**Discussion by PilgrimToHyperion** [Pryaxis/TShock#2128] [General]
Just about to kill WOF and then my game updated! How long does it normally take for TShock to be updated? Don't crucify me for asking please! :P

**Comment by WiIIiam278** [comment]
I, too, have this question. Unfortunately, it seems like it will take quite a while for things to happen (I run on a Pterodactyl deployment and the eggs available for that are either vanilla up to 1.4.0.2 or tShock). Until then, there's no real options apart from getting players to install [GameLauncher](https://forums.terraria.org/index.php?threads/how-to-legally-play-old-versions-of-terraria-even-up-to-1-0-0.70200/) or something to downgrade (which is a hassle). It's a shame they don't provide a way of downgrading easily.

**Comment by collinshg** [comment]
I'm also curious. I had an open Terraria server for twitch viewers and all their characters (and stuff) are gone until TShock is compatible with the new update. Usually new members join on Thursdays when I stream it, so what are the chances it will work by then?

**Comment by kittenchilly** [comment]
Been searching and there's barely been any activity, so I think its justified to ask this. I hope a 1.4.1 version comes soon.

**Comment by hakusaro** [comment]
Sorry, we don’t have a NordVPN sponsorship here. We’re operating on
table scraps. Just look at how many GitHub sponsors we have!

On October 14, 2020, GitHub <notifications@github.com> wrote:
> Been searching and there's barely been any activity, so I think its
> justified to ask this. I hope a 1.4.1 version comes soon.
>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Pryaxis/TShock/discussions/2128#discussioncomment-

98327>, or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAECBJ2ODNA46JAOC5OOQUDSKXXKFANCNFSM4SPPX7PA>.

**Comment by hakusaro** [comment]
> Don't crucify me for asking please! :P

Not funny!

**Comment by hakusaro** [comment]
> Usually new members join on Thursdays when I stream it, so what are the chances it will work by then?

The odds increase with the amount of love you throw at valuable team members @Patrikkk and @Quinci135.

**Comment by hakusaro** [comment]
Wait, didn't you all see this thread? https://github.com/Pryaxis/TShock/discussions/2125
